mobile/src/main/java/org/openhab/habdroid/ui/ItemPickerActivity.kt

```diff
@@ -91,7 +91,7 @@ class ItemPickerActivity : AbstractBaseActivity(), SwipeRefreshLayout.OnRefreshL
 
         if (!getPrefs().isTaskerPluginEnabled()) {
             isDisabled = true
-            updateViewVisibility(false, false, true)
+            updateViewVisibility(loading = false, loadError = false, isDisabled = true)
```

Comment:
AS is printing the parameter name next to the arguments, if these are primitive types. So IMO this change isn't really needed.

Reply:
There's a style inspection that emits a warning if there's multiple unnamed boolean parameters for one method, presumably to avoid them being confused.